src/Avalonia.Controls/Slider.cs

```diff
@@ -81,57 +121,83 @@ public double TickFrequency
             set { SetValue(TickFrequencyProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets a value that indicates where to draw 
+        /// tick marks in relation to the track.
+        /// </summary>
+        public TickPlacement TickPlacement
+        {
+            get { return GetValue(TickPlacementProperty); }
+            set { SetValue(TickPlacementProperty, value); }
+        }
+
         /// <inheritdoc/>
         protected override void OnApplyTemplate(TemplateAppliedEventArgs e)
```

Comment:
I think we should call `base.OnApplyTemplate` here as well. In case somebody add something to `RangeBase` and we break something by not letting it initialize.